platforms/hyperledger-fabric/configuration/README.md

```diff
@@ -68,7 +68,7 @@ We are working on a verification document for Hyperledger Fabric.
     ```
     ansible-playbook add-organization.yaml -e "@/path/to/network-fabric.yaml" -e "add_new_org='true'"
     ```
-    Follow steps in [README](https://github.com/hyperledger-labs/blockchain-automation-framework/docs/source/operations/adding_new_org_fabric.md).
+    Follow steps in [README](https://github.com/hyperledger-labs/blockchain-automation-framework/blob/develop/docs/source/operations/adding_new_org_fabric.md).
```

Comment:
Add readthedocs link instead

Reply:
Changed in [c44e8bf](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/1096/commits/c44e8bfc500eb45912b5fb9988085bb0eb99e50c). I also updated the readthedocs link to add the `-e "add_new_org=true"` flag to the command.

Reply:
See comment below for completion